nautobot/ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "nautobot",
-    "version": "2.1.0-b1",
+    "version": "2.1.0",
```

Comment:
```suggestion
    "version": "2.1.0-b1",
```

Reply:
We'll bump the version after we cut the release. And we'll bump it to beta 2, not straight to 2.1.0